test/PoolManager.t.sol

```diff
@@ -1002,8 +1000,9 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
     function test_collectProtocolFees_ERC20_accumulateFees_gas() public {
         uint256 expectedFees = 7;
 
-        feeController.setProtocolFeeForPool(key.toId(), MAX_FEE_BOTH_TOKENS);
-        manager.setProtocolFee(key);
+        uint24 protocolFee = MAX_FEE_BOTH_TOKENS;
+        vm.prank(address(feeController));
```

Comment:
and same with all the tests for feeController - better to code the feeController to call the manager